mtk.go

```diff
@@ -0,0 +1,25 @@
+package nmea
+
+const (
+	// TypeMTK type for PMTK sentences
+	TypeMTK = "PMTK"
```

Comment:
I missed this the first time around, but "P" is special talker meaning "proprietary". "MTK" the message type (like the `MTK` struct name). So this should just be "MTK" instead of "PMTK".

Reply:
https://www.rhydolabz.com/documents/25/PMTK_A11.pdf

according to the pdf doc (page 5) Talker ID is `PMTK` and the Type are the next 3 bytes(some int numbers)

if we set MTK as the type, what would be the next 3 bytes?

Reply:
Looks like your right. ... sigh ... why is nmea so inconsistent 😢